src/scss/atomic/molecules/_main-nav.scss

```diff
@@ -66,7 +66,7 @@ $mainNavBorder: 1px solid $colorGray;
     .main-nav--small & {
         position: relative;
         top: 100%;
-        left: 0;
+        left: -100%;
```

Comment:
?

Reply:
Is part of the header. When you resize the window the header is animated. I have applied also other code to fix it.

Reply:
Nice ;-) that was annoying!